src/Result.cpp

```diff
@@ -0,0 +1,17 @@
+//
+// Created by Tianyi Wang on 5/14/23.
+//
+
+#include "../include/Result.hpp"
```

Comment:
This is not how to include project headers. If everything is set up properly, then
```suggestion
#include "Result.hpp"
```
is enough and the way to go.

Reply:
Thanks for clearing this up, but my CLion gives error messages (although they probably don't affect the build) after I remove this, although I included all files into CMake. Do you think I missed something here?

Reply:
If everything is set up properly, this should not give any error messages. I'll check again.